### PR TITLE
Television category/features and box rows

### PR DESCRIPTION
**Pull request by atrovato**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- ~~[ ] If your changes affects code, did your write the tests?~~
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- ~~[ ] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to [the community](https://community.gladysassistant.com/) for testing before merging.~~
- ~~[ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)~~
- ~~[ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).~~
- [x] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

image

### CODE REVIEW

**Comment by Pierre-Gilles** [comment]
Looks very nice!

File: front/src/components/boxs/device-in-room/device-features/NumberDeviceFeature.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import get from 'get-value';
+
+import { getDeviceName } from './utils';
+import { DeviceFeatureCategoriesIcon } from '../../../../utils/consts';
+
+const MultiLevelDeviceType = ({ children, ...props }) => {
```
Comment (Pierre-Gilles) [inline front/src/components/boxs/device-in-room/device-features/NumberDeviceFeature.jsx]: I guess it should be NumberDeviceFeature here :)

File: server/utils/constants.js [hunk 2]
```diff
@@ -375,6 +376,35 @@ const DEVICE_FEATURE_TYPES = {
...
+    VOLUME_UP: 'volume-up',
+    VOLUME_DOWN: 'volume-down',
+    VOLUME_MUTE: 'volume-mute',
+    VOLUME: 'volume',
+    PLAY: 'play',
+    PAUSE: 'pause',
+    STOP: 'stop',
+    REWIND: 'rewind',
+    FORWARD: 'forward',
+    RECORD: 'record',
```
Comment (Pierre-Gilles) [inline server/utils/constants.js:407]: How are all of those features handled? Not handled yet right?

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Just a few feedbacks! Looks nice!

File: front/src/components/boxs/device-in-room/device-features/NumberDeviceFeature.jsx (hunk 1 above)
Comment (atrovato) [reply front/src/components/boxs/device-in-room/device-features/NumberDeviceFeature.jsx]: No, I didn't copy/paste :p

File: server/utils/constants.js (hunk 2 above)
Comment (atrovato) [reply server/utils/constants.js:407]: Those are not visible on dashboard, at least not yet, so not actionnable for now.
But incoming broadlink (IR remote) service will allow to use them.

For dashboard presentation, let's think about a nice display.

Comment (Pierre-Gilles) [reply server/utils/constants.js:407]: Yes sure, just wanted to know if it was used or not :)

**Comment by Pierre-Gilles** [comment]
PR is nice, but the demo mode looks a bit "messy" now, demo mode is supposed to be a "promo" mode, not really the testing mode :D 

Maybe we could either put those devices in another box on the dashboard? (like on the right there is some space), or it could be another dashboard if it's too much here

Screenshot 2021-12-03 at 12 49 28

**Comment by atrovato** [comment]
What about ?
image

**Comment by Pierre-Gilles** [comment]
Looks way better to me! 👍

**Review by Pierre-Gilles — APPROVED** [review]

**Pull request merged**